bayeux.go

```diff
@@ -81,6 +81,14 @@ type clientIDAndCookies struct {
 	cookies  []*http.Cookie
 }
 
+type AuthenticationParameters struct {
+	ClientID     string
+	ClientSecret string
+	Username     string
+	Password     string
+	TokenURL     string
```

Comment:
Would you mind adding any comments here? Maybe also sample values, so the library user will know what kind of value the function expects.

Reply:
Makes sense, updated the same. Thanks!